### ISSUE TITLE
[Packager] Source-map explicitly enabled, but lost when --minify is enabled

### ISSUE THREAD

**Issue by paramaggarwal** [facebook/react-native#3527]
TLDR: I enabled source maps in `node_modules/react-native/local-cli/bundle.js` by explicitly setting `inlineSourceMap: true,` in the code. Then running `react-native bundle --out js.bundle` works, but `react-native bundle --minify --out js.bundle` loses the source maps. I am looking to generate a source map between the original files and the minified bundle.

We use a minified bundle in production and send the error string to a server to accumulate errors (Crashlytics). The problem at the moment is that the error message is not helpful as it is completely minified code without source maps.

Is there a way to hand over a separate source-maps file in production, or be able to keep a source-map file on the server so that the server could link the error to the correct place in code? Anyone who has done this?

I was looking around and something like this looks useful: https://errorception.com/docs/sourcemaps (it fetches source maps on the remote server to help with debugging.)

**Comment by jsierles** [comment]
Looks like this was fixed!

**Comment by paramaggarwal** [comment]
Thanks @jsierles - Yes, it was! I forgot the close this when I closed the PR.
